xml/admin_gui_dashboard.xml

```diff
@@ -2327,6 +2410,130 @@ Oct 04 07:48:53 doc-ses6min4 systemd[1]: Started Ceph rbd mirror daemon.
+     <para>
+       The following procedure needs to be followed once to create the Root CA.
+     </para>
+     <procedure>
+       <step>
+         <note>
+           <para>
+             This is the key used to sign the certificate requests, anyone
+             holding this can sign certificates on your behalf.
+           </para>
+         </note>
+         <para>
+           Create the Root Key:
+         </para>
+<screen>
+openssl genrsa -des3 -out rootCA.key 4096
+</screen>
+        <note>
+          <para>
+            If you want a non-password protected key, remove the
+            <option>-des3</option> option.
+          </para>
+        </note>
+       </step>
+       <step>
+         <para>
+           Create and self-sign the Root Certificate:
+         </para>
+<screen>
+openssl req -x509 -new -nodes -key rootCA.key -sha256 -days 1024 -out rootCA.crt
+</screen>
+       </step>
+     </procedure>
+     <para>
+       The following procedure needs to be followed for each server
+       that needs a trusted certificate from our CA.
+     </para>
+     <procedure>
+       <step>
+         <para>
+          Create the certificate key:
+        </para>
+<screen>
+openssl genrsa -out mydomain.com.key 2048
+</screen>
+       </step>
+       <step>
+         <para>
+           The certificate signing request is where you specify the details for
+           the certificate you want to generate. This request is processed by
+           the owner of the Root Key to generate the certificate.
+         </para>
+         <important>
+           <para>
+             When creating the signing request, it is important to specify
+             the <literal>Common Name</literal> providing the IP address or
+             domain name for the service, otherwise the certificate cannot
+             be verified.
+           </para>
+         </important>
+         <para>
+           These are two ways to create the CSR:
+         </para>
+         <itemizedlist>
+           <listitem>
+             <para>
+               Interactive method:
+             </para>
+<screen>
+openssl req -new -key mydomain.com.key -out mydomain.com.csr
+</screen>
+           </listitem>
+           <listitem>
+             <para>
+               One-liner method:
+             </para>
+<screen>
+openssl req -new -sha256 -key mydomain.com.key -subj "/C=US/ST=CA/O=MyOrg, Inc./CN=mydomain.com" -out mydomain.com.csr
+</screen>
+             <para>
+               If you need to pass additional config you can use
+               the <option>-config</option> parameter. For example:
+             </para>
+<screen>
+  openssl req -new -sha256 \
+      -key mydomain.com.key \
+      -subj "/C=US/ST=CA/O=MyOrg, Inc./CN=mydomain.com" \
+      -reqexts SAN \
+      -config &lt;(cat /etc/ssl/openssl.cnf \
+          &lt;(printf "\n[SAN]\nsubjectAltName=DNS:mydomain.com,DNS:www.mydomain.com")) \
+      -out mydomain.com.csr
+</screen>
+           </listitem>
```

Comment:
I would expect here a short `para` explaining that the command line options take precedence over something defined in the config file.